portal_spa/src/components/layout/AttributeIndexLayout.tsx

```diff
@@ -0,0 +1,209 @@
+import { useEffect, useState } from "react";
+import { Outlet, Link, useMatch } from "react-router-dom";
+import { axiosInstance } from "@/services/axiosInstance";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import { ScrollArea } from "@/components/ui/scroll-area";
+import { Input } from "@/components/ui/input";
+import type { Credential } from "@/models/credential";
+import { useLocation } from "react-router-dom";
+import { Separator } from "../ui/separator";
+
+export default function AttributeIndexLayout() {
+  const [credentials, setCredentials] = useState<Credential[]>([]);
+  const [searchQuery, setsearchQuery] = useState("");
```

Comment:
Use consisting casing

```suggestion
  const [searchQuery, setSearchQuery] = useState("");
```